json2html/jsonconv.py

```diff
@@ -23,6 +27,9 @@ def convert(self,**args):
 		'''
 		convert json Object to HTML Table format
 		'''
+		use_bootstrap = False
```

Comment:
It would reduce the number of pass_ons for this variable if this can be made global just like `a` : `use_bootstrap = False`. Please make it global and remove it as a parameter from each fn declaration, wherever you need to use `use_bootstrap` just write `global use_bootstrap` and then use it in next line. Hope that makes sense, right? :smile: